cli/info.go

```diff
@@ -0,0 +1,220 @@
+package cli
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/fatih/color"
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/go-fil-markets/storagemarket"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+var infoCmd = &cli.Command{
+	Name:   "info",
+	Usage:  "Print node info",
+	Action: infoCmdAct,
+}
+
+func infoCmdAct(cctx *cli.Context) error {
+	fullapi, acloser, err := GetFullNodeAPIV1(cctx)
+	if err != nil {
+		return err
+	}
+	defer acloser()
+	ctx := ReqContext(cctx)
+
+	network, err := fullapi.StateGetNetworkParams(ctx)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Network: %s\n", network.NetworkName)
+	fmt.Print("Chain: ")
+	head, err := fullapi.ChainHead(ctx)
+	if err != nil {
+		return err
+	}
+
+	switch {
+	case time.Now().Unix()-int64(head.MinTimestamp()) < int64(build.BlockDelaySecs*3/2): // within 1.5 epochs
+		fmt.Printf("[%s]", color.GreenString("sync ok"))
+	case time.Now().Unix()-int64(head.MinTimestamp()) < int64(build.BlockDelaySecs*5): // within 5 epochs
+		fmt.Printf("[%s]", color.YellowString("sync slow (%s behind)", time.Now().Sub(time.Unix(int64(head.MinTimestamp()), 0)).Truncate(time.Second)))
+	default:
+		fmt.Printf("[%s]", color.RedString("sync behind! (%s behind)", time.Now().Sub(time.Unix(int64(head.MinTimestamp()), 0)).Truncate(time.Second)))
+	}
```

Comment:
If this is duplicated with the code in lotus-miner info, we should put this in a function here, and call that from lotus-miner to not duplicate logic.

Reply:
Its duplicated with the code in `lotus-miner info` so I put it in a function here: https://github.com/filecoin-project/lotus/pull/9233/commits/818153a1e71a14f913572f1b5659fb8ad250b4af

Was able to call it on the `lotus-miner info` side as well but for that I needed to `GetFullNodeAPIV1` on that side. I might be missing some understanding between v0api and v1api, so might bug you a bit during a standup session.